### DISCUSSION TITLE
How to plot matrix data?

### DISCUSSION THREAD

**Discussion by zhifeiji-han** [SCIInstitute/SCIRun#2308] [Q&A]
I have a matrix data, which  dimension is 3 * 21, 3 represents three sensors, 21 represents 21 voltage data . 
How can this matrix display the voltage distribution image of each sensor in scirun, as shown in the figure below?
![image](https://user-images.githubusercontent.com/70640243/125016519-123a4880-e0a4-11eb-9483-82fcc19406b5.png)

**Comment by jab0707** [comment]
There is a line plotter module that can take in vector data. I would use getmatrix slice then lineplotter individually for each. I am not sure, but I think line plotter can also take matrix data, it just might need to be transposed depending on if your data is column based or row based. In the case that you need it transposed, the evaluateLinearAlgebraUnary module will do that.
